e2e/test/scenarios/dashboard/tabs.cy.spec.js

```diff
@@ -410,3 +467,19 @@ describeWithSnowplow("scenarios > dashboard > tabs", () => {
     );
   });
 });
+
+/**
+ * When you need to postpone a response (to check for loading spinners or alike),
+ * use this:
+ *
+ * `cy.intercept('POST', path, delayResponse(1000)).as('delayed')`
+ *
+ * `cy.wait('@delayed')` - you'll have 1000 ms until this resolves
+ */
+function delayResponse(delayMs) {
```

Comment:
As discussed with @nemanjaglumac, we don't extract this as a global helper until needed by some other test.